### PR TITLE
refactored images-constructor page to be localised with i18next library

### PR DESCRIPTION
**Pull request by boris-giga**
## Description

Applied i18next.js for images-constructor page localisation

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅All tests passed locally
- [ ] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/locales/en/constructor.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "titles": {
+    "model": "Model",
+    "basis": "Basis",
+    "pattern": "Pattern",
+    "bottom": "Bottom",
+    "size": "Size",
+    "defaultPrice": "Default item price:",
+    "totalPrice": "Total item price:",
+    "endPrice": "End price:",
+    "defaultPriceValue": "50 ",
```
Comment (maarekua) [inline src/locales/en/constructor.json:11]: Cant find usage of "defaultPriceValue"

File: src/locales/en/constructor.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "titles": {
+    "model": "Model",
+    "basis": "Basis",
+    "pattern": "Pattern",
+    "bottom": "Bottom",
```
Comment (maarekua) [inline src/locales/en/constructor.json:6]: Some translations have a common use, move it to some general translation file like common.json, discuss it with your teammates they have the same comments.

Comment (maarekua) [reply src/locales/en/constructor.json:6]: Words like Size, Bottom, Model, Submit, Cancel, look through your translations. Thanks,

File: src/locales/ua/constructor.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "titles": {
+    "model": "Модель",
+    "basis": "Основа",
+    "pattern": "Гобелен",
+    "bottom": "Низ",
+    "size": "Розмір",
+    "defaultPrice": "Базова ціна товару:",
+    "totalPrice": "Загальна вартість",
+    "endPrice": "Кінцева ціна:",
+    "defaultPriceValue": "1400 ",
```
Comment (maarekua) [inline src/locales/ua/constructor.json:11]: No usage of defaultPriceValue, also check other translations. Thanks.

File: src/locales/ua/constructor.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "moreOptions": "Більше опцій"
+  },
+  "basicErrors": {
+    "basisNotFound": "Основа Не Знайдена!",
+    "basisAlreadyExist": "Така основа вже існує!",
+    "imageNotProvided": ""
```
Comment (maarekua) [inline src/locales/ua/constructor.json:17]: I think empty translation is redunant.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/locales/en/constructor.json (hunk 1 above)
Comment (boris-giga) [reply src/locales/en/constructor.json]: in that case, I'll remove this file and spread translations into "common.json" and "buttons.json" with removing redundant translatons

File: src/pages/images-constructor/tests/images-constructor.spec.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+const themeValue = theme('light');
+
+let wrapper;
+
+jest.mock('horondi_merge_images', () => ({ DEFAULT_PRICE_VALUE: '1400' }));
+
+describe('ImagesConstructor component tests', () => {
+  wrapper = shallow(
+    <ThemeProvider theme={themeValue}>
```
Comment (maarekua) [inline src/pages/images-constructor/tests/images-constructor.spec.js:18]: Mock useStyles and then you can remove ThemeProvider. Thanks.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/pages/images-constructor/tests/images-constructor.spec.js (hunk 5 above)
Comment (boris-giga) [reply src/pages/images-constructor/tests/images-constructor.spec.js:14]: applied for constructor-submit.spec.js , in current file this causes issues with wrapping with <Provider>, can't find solution

**Review by olga-gchak — APPROVED** [review]

**Review by olga-gchak — APPROVED** [review]

**Pull request closed without merging**